### DISCUSSION TITLE
跨组件触发请求 是否只支持 get 请求?

### DISCUSSION THREAD

**Discussion by vaynevayne** [alovajs/alova#127] [Q&A]
文档没说明, 默认是支持所有请求对吗? 
为什么我会这样问, 因为 react-query 的 revaliate 重试, 只支持 get 请求, 但是文档没写

**Comment by JOU-amjs** [comment]
你说的是[这个跨组件触发请求](https://alova.js.org/zh-CN/strategy/actionDelegationMiddleware)吗？这个是没有限制的。

如果你问的是数据缓存相关的问题，alova的get请求默认会有5分钟的内存缓存时间，你也可以为不同的请求方式设置缓存时间，[链接在这边](https://alova.js.org/zh-CN/learning/response-cache/#%E5%85%A8%E5%B1%80%E8%AE%BE%E7%BD%AE%E7%BC%93%E5%AD%98%E6%A8%A1%E5%BC%8F)

**Reply by vaynevayne** [reply]
那你这个挺赞的
react-query 的跨组件请求和缓存做一起的, 所以根本分不开
